backend/package.json

```diff
@@ -4,10 +4,10 @@
   "description": "",
   "main": "dist/app.js",
   "scripts": {
-    "start:dev": "npx tsc && NODE_ENV=development node build/app.js",
+    "start:dev": "npx tsc && SET NODE_ENV=development node build/app.js",
```

Comment:
Please exclude changes to the `package.json` from the PR, it will break the linux workflow, and it's outside of the scope.

Reply:
Alright, Thanks

> Please exclude changes to the `package.json` from the PR, it will break the linux workflow, and it's outside of the scope.